src/main/java/net/snowflake/client/core/SessionUtil.java

```diff
@@ -1282,6 +1282,10 @@ static void updateSfDriverParamValues(Map<String, Object> parameters, SFBaseSess
         if (session != null) {
           session.setEnableHeartbeat((Boolean) entry.getValue());
         }
+      } else if (CLIENT_SESSION_KEEP_ALIVE_HEARTBEAT_FREQUENCY.equalsIgnoreCase(entry.getKey())) {
```

Comment:
In this function, we assign the values from the connection parameters to their corresponding variables in SFSession/SFBaseSession.